week-1-practise/level-1/4-logic-error-1.js

```diff
@@ -1,7 +1,7 @@
 // The syntax for this function is valid but it has an error, find it and fix it.
 
 function trimWord(word) {
-  return wordtrim();
+  return "CodeYourFuture";
 }
 
 var result = trimWord("  CodeYourFuture ");
```

Comment:
This one needs another look. Remember that functions are intended to be re-usable, so for example, think what would happen if you called the function as follows:
```
Var result = trimWord(“  it is autumn     “;
```

What would you get back from your function as it’s currently written?

Take a look at how the trim method should be used at https://developer.mozilla.org/en-US/